package.json

```diff
@@ -52,13 +52,13 @@
     "@typescript-eslint/eslint-plugin": "^6",
     "@typescript-eslint/parser": "^6",
     "aws-cdk": "2.139.0",
-    "esbuild": "^0.20.2",
+    "esbuild": "^0.21.1",
     "eslint": "^8",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.1",
     "jest": "^29.7.0",
     "jest-junit": "^15",
-    "projen": "^0.81.2",
+    "projen": "^0.81.8",
```

Comment:
Consider upgrading the `projen` dependency to the latest version `0.81.8` to take advantage of the latest bug fixes and improvements. [medium]